samples/nrf9160/aws_fota/src/main.c

```diff
@@ -448,13 +450,32 @@ static void at_configure(void)
 	__ASSERT(err == 0, "AT CMD could not be established.");
 }
 
-static void aws_fota_cb_handler(enum aws_fota_evt_id evt)
+static void aws_fota_cb_handler(struct aws_fota_event *fota_evt)
 {
 	int err;
 
-	switch (evt) {
+	if (fota_evt == NULL) {
```

Comment:
Instead of this you can add a default case to the switch case which should catch this

Reply:
the switch statement is accessing the `id` field in the `fota_evt` pointer.